.github/workflows/indent.yml

```diff
@@ -2,23 +2,23 @@ name: Indent
 
 on:
   push:
-    branches: [ master, develop ]
+    branches: [master, develop]
   pull_request:
-    branches: [ master, develop ]
+    branches: [master, develop]
 
 jobs:
   build:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - uses: DoozyX/clang-format-lint-action@v0.11
-      with:
-        source: '.'
-        exclude: ''
-        extensions: 'h,cc'
-        clangFormatVersion: 8
-        inplace: true
-    - run: |
-        git diff
-        git diff-files --quiet || exit $?
+      - uses: actions/checkout@v2
+      - uses: DoozyX/clang-format-lint-action@v0.11
+        with:
+          source: "."
+          exclude: "./src/json.hpp"
+          extensions: "hpp,cpp,c,h"
+          clangFormatVersion: 8
+          inplace: false
```

Comment:
Why changing inplace here? Compaing the diff without inplace doesn't make sense, right?

Reply:
If there is a change required you can actually read from log but fine to keep as inplace